src/js/components/VoterGuide/OrganizationVoterGuideCandidateList.jsx

```diff
@@ -18,7 +18,18 @@ export default class OrganizationVoterGuideCandidateList extends Component {
               contest_office_name={this.props.contest_office_name}
               link_to_ballot_item_page
               organization_we_vote_id={this.props.organization_we_vote_id}
-              {...child}
+              ballotItemDisplayName={child.ballot_item_display_name}
```

Comment:
Thank you @fdiazq, these changes look correct. You will also need to update the variables names coming into `OrganizationVoterGuideCandidateItem.propTypes` in /src/js/components/VoterGuide/OrganizationVoterGuideCandidateItem, and make sure to convert all uses of these incoming prop variable names to camelCase. While you are working on this, please update the remaining incoming snake_case variables to camelCase (ex/ link_to_ballot_item_page).